### PR TITLE
feat: Updating some dependencies

### PR DESCRIPTION
**Pull request by YosafatEG**
# Updating some dependencies

Bump eslint from 7.32.0 to 8.33.0
Bump commitizen from 4.2.6 to 4.3.0
Bump tape from 5.6.1 to 5.6.3
Bump @hapi/topo from 5.1.0 to 6.0.0
Bump @hapi/hoek from 9.3.0 to 11.0.2
Bump dot-prop from 5.2.0 to 6.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,9 +29,9 @@
     }
   },
   "dependencies": {
-    "@hapi/hoek": "^9.0.4",
-    "@hapi/topo": "^5.0.0",
-    "@vrbo/determination": "^6.0.1",
+    "@hapi/hoek": "^11.0.2",
+    "@hapi/topo": "^6.0.0",
+    "@vrbo/determination": "^6.1.3",
```
Comment (shellbj) [inline package.json:34]: this range doesn't need to move

File: package.json [hunk 2]
```diff
@@ -42,13 +42,13 @@
     "@hapi/hapi": "^20.2.1",
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/git": "^10.0.1",
-    "commitizen": "^4.2.4",
+    "commitizen": "^4.3.0",
     "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^7.7.0",
+    "eslint": "^8.33.0",
```
Comment (shellbj) [inline package.json:47]: `@hapi/eslint-config-hapi` and ``@hapi/eslint-plugin-hapi`  are deprecated and moved to `@hapi/eslint-plugin`; this should be part of this update for `eslint`

File: package.json [hunk 3]
```diff
@@ -29,9 +29,9 @@
     }
   },
   "dependencies": {
-    "@hapi/hoek": "^9.0.4",
-    "@hapi/topo": "^5.0.0",
-    "@vrbo/determination": "^6.0.1",
+    "@hapi/hoek": "^11.0.2",
+    "@hapi/topo": "^6.0.0",
```
Comment (shellbj) [inline package.json:33]: Upgrading both of these will cause conflicts in other packages that consume this one.  

Minimally, they should be the lowest, newest, version possible (`^11.0.0` vs. `^11.0.2`); but I'm tempted to say we should use a range expansion (`^9.0.4 || ^10.0.0 || ^11.0.0`) on these until `hapi` upgrades can be rolled out across the product line.

**Review by shellbj — CHANGES_REQUESTED** [review]

File: package.json [hunk 4]
```diff
@@ -29,26 +29,25 @@
     }
   },
   "dependencies": {
-    "@hapi/hoek": "^9.0.4",
-    "@hapi/topo": "^5.0.0",
+    "@hapi/hoek": "^9.0.4 || ^10.0.0 || ^11.0.0",
```
Comment (kisaiev) [inline package.json:32]: `@hapi/hoek` at [v11.0.0](https://www.npmjs.com/package/@hapi/hoek/v/11.0.0) is deprecated, so it's better to start with `^11.0.1` in v11 range.

File: .github/dependabot.yml [hunk 5]
```diff
@@ -13,3 +13,10 @@ updates:
     reviewers:
       - "@ExpediaGroup/catalyst-committers"
     open-pull-requests-limit: 10
+    ignore:
+      # Ignore Dot Prop major updates
+      - dependency-name: "dot-prop"
+        update-types: ["version-update:semver-major"]
+      # Ignore Semantic Release major updates
+      - dependency-name: "semantic-release"
+        update-types: ["version-update:semver-major"]
```
Comment (shellbj) [inline .github/dependabot.yml:22]: Include `hapi` here

**Review by shellbj — APPROVED** [review]

**Review by kisaiev — APPROVED** [review]

**Pull request merged**